docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md

```diff
@@ -69,8 +69,8 @@ spec:
       type: CLOUDRUN
       config:
         project: gcp-project
-        region: cloudrun-region
-        credentialsFile: path-to-the-service-account-file
+        region: {CLOUDRUN_REGION}
```

Comment:
I would prefer `YOUR_CLOUDRUN_REGION`, but up to you

Reply:
To be honest, I don't like that `YOUR` prefix. By adding it we will have `YOUR_REGION`, `YOUR_PROJECT`,  `YOUR_CREDENTIAL_FILE`...

Reply:
🤣 I have the same sense as @nghialv

Reply:
I hear you. `{}` and capitalization are enough representing they are placeholders 👍